alcs-api/src/application/application.controller.ts

```diff
@@ -153,7 +157,18 @@ export class ApplicationController {
       highPriority: application.highPriority,
     });
 
-    const mappedApps = await this.applicationService.mapToDtos([app]);
+    if (app.assigneeUuid !== existingApplication.assigneeUuid) {
```

Comment:
What if users mentioned themselves?

Reply:
Good idea, added logic + unit test to NOT create a notification if user is assigning to themselves